Sources/SwiftFormat/Core/RuleRegistry+Generated.swift

```diff
@@ -33,6 +33,7 @@
     "NoAssignmentInExpressions": true,
     "NoBlockComments": true,
     "NoCasesWithOnlyFallthrough": true,
+    "NoEmptyLinesOpeningClosingBraces": true,
```

Comment:
This will change the behavior of the default configuration (and also any existing configuration that doesn't explicitly reference the new rule). Is that something you intend to do with this PR?

Adding `public override class var isOptIn: Bool { return true }` would be safer for now. Then, we should probably come up with a policy for how we want to handle changes to the default configuration in the future.

Reply:
Sorry, It wasn’t my intention.